CMakeLists.txt

```diff
@@ -100,6 +100,11 @@ endif()
 option(WRAP_PYTHON "Generate a pybind11 wrapper" OFF)
 if (WRAP_PYTHON)
       INCLUDE_DIRECTORIES(SYSTEM pybind11)
+      # There is some problem with LTO on mingw
+      if (WIN32)
+	      set(PYBIND11_LTO_CXX_FLAGS "")
+      endif()
+      #add_custom_command(TARGET neml POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy 
```

Comment:
do we need this?

Reply:
Yes, unfortunately we need that.  That's the workaround for the actual compiler bug  I whined about.  Or if you're talking about the commented-out command, that is a mistake I need to fix.